Formula/g/gost.rb

```diff
@@ -28,11 +18,7 @@ class Gost < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "ccbbcfddd4e05137303c86f5d25bbbf8ecccd093a72c6af80cd79b800bdf5d4b"
   end
 
-  # no release to support go1.20, https://github.com/ginuerzh/gost/issues/1012
-  # also no actions on go1.21 build support PR, https://github.com/ginuerzh/gost/pull/983
-  deprecate! date: "2024-02-14", because: :unmaintained
-
-  depends_on "go@1.20" => :build
+  depends_on "go@1.22" => :build
```

Comment:
gost is using Go 1.23 for building:
https://github.com/ginuerzh/gost/blob/87d6a2fdc2cc04fa6f7c90e6cc9e8406076f6928/Dockerfile#L3

so it could easily use current Go here as well:
```suggestion
  depends_on "go" => :build
```

Reply:
Perhaps versions could be unified upstream, because it is using Go 1.22 for releases:
https://github.com/ginuerzh/gost/blob/87d6a2fdc2cc04fa6f7c90e6cc9e8406076f6928/.github/workflows/release.yml#L24